src/gateway/tasks/contractsMetadata.ts

```diff
@@ -96,6 +96,7 @@ async function loadContractsFromGql(context: GatewayContext) {
       contractsInserts.push({
         contract_id: transaction.node.id,
         block_height: transaction.node.block.height,
+        block_timestamp: transaction.node.block.timestamp,
```

Comment:
I believe the block.timestamp is not currently specified in the GQL...

Reply:
oops, I was so sure about it I haven't even tested it :)) now its fine!